proxy/src/http/sql_over_http.rs

```diff
@@ -223,7 +247,13 @@ pub async fn handle(
                 }
             }
             transaction.commit().await?;
-            Ok(json!({ "results": results }))
+            Ok(json!({
+                "results": results,
+                "options": {
+                    "isolationLevel": txn_isolation_level.map(|l| format!("{:?}", l)),
```

Comment:
in JSON, we usually follow `snake_case` (checked public API and Pageserver spec)

Reply:
resolved in 3e28542f47a9579b6f399980816c0562f32bf5a3 (GitHub has some delays processing PR commits -- not showing up in the web page)